tests/terratest/functions/set_modules/setHostedModules.go

```diff
@@ -0,0 +1,35 @@
+package functions
```

Comment:
The name of this file doesn't match what was set for non-hosted modules. Can you update either this name to not have `modules` in the filename, or update the others to have `modules` in their filenames?

Reply:
These names can be updated.

The functions to set the actual module itself will not include `module` in the function name (e.g. `SetAKS()` )

For context, SetConfigTF() is passed in a module, and depending on that module, will call either SetHostedModules() or SetNodeDriverModules()... and they do just that.. SetHostedModules() takes in the module passed in from SetConfigTF()  and depending on that value, will call either SetAKS(), SetEKS(), or SetGKE().  SetNodeDriverModules() do the same, but for Node driver clusters.

`SetConfigTF()`:
- `SetHostedModules()`:
  - `SetAKS()`
  - `SetEKS()`
  - `SetGKE()`
- `SetNodeDriverModules()`:
  - `SetEC2RKE1()`
  - `SetEC2RKE2K3s()`
  - `SetLinodeRKE1()`
  - `SetLinodeRKE2K3s()`   
  
I'm open to changing the names here, and based on your input, would agree we could drop `Modules` for these two functions, and just have them SetHosted() and SetNodeDriver().. although, in a way, its less specific to what the function is actually doing.

Maybe SetHostedModule() and SetNodeDriverModule() would be most specific, as they only set one module at a time 🤔 

Thoughts on naming convention here?

Reply:
ok I see.  It makes sense when you lay it out like that. But the file structure doesn't match the function bullets that you laid out, which is why I was confused. everything is in 1 `set_modules` folder, so it was hard to tell that those do different things from looking at it. But this might be fine as-is, lets get another round of review before changing anything.